hooks/scan_scene_tk-houdini.py

```diff
@@ -65,4 +60,127 @@ def execute(self, **kwargs):
 
         # create the primary item - this will match the primary output 'scene_item_type':
         items.append({"type": "work_file", "name": name})
+
+        # look for alembic caches to publish
+        items.extend(self._get_alembic_items())
+
+        # look for rendered images to publish
+        items.extend(self._get_rendered_image_items())
+
         return items
+
+    def _get_alembic_items(self):
+        """Scan the file for tk alembic nodes with already exported caches."""
```

Comment:
Should update the docstring to reflect the fact that non-exported caches'll show up as well.